### PR TITLE
chore: change repository owner to team-apm

### LINKED ISSUE #321: Organization
## Is your feature request related to a problem? Please describe.

An organization has more functions than a personal repository.
I think that if this project becomes bigger, I should make an organization.

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Change repository owner in `package.json` and `forge.config.js` to team-apm.
- Fix an error around auto-update.

## Related Issue

close #321

## Motivation and Context

Refer to #321 

## How Has This Been Tested?

- Checked the feature around the auto-update.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -153,13 +172,13 @@ function checkUpdate(silent = true) {
 }
 
 const doAutoUpdate = store.get('autoUpdate');
-if (!isDevEnv && typeof doAutoUpdate === 'string') {
-  if (doAutoUpdate === 'download') {
-    updateElectronApp();
-  } else if (doAutoUpdate === 'notify') {
-    checkUpdate();
-  }
+// if (!isDevEnv && typeof doAutoUpdate === 'string') {
```
Comment (mitosagi) [inline src/main.ts:175]: For debugging?

**Review by mitosagi — COMMENTED** [review]
There is one thing to check.

File: src/main.ts (hunk 1 above)
Comment (hal-shu-sato) [reply src/main.ts:175]: Oh...
That's right...

**Comment by hal-shu-sato** [comment]
I fixed it.

**Review by mitosagi — APPROVED** [review]
It looks good to me.

**Pull request merged**